src/apps/Profile/ipfs/index.js

```diff
@@ -0,0 +1,9 @@
+import ipfsClient from 'ipfs-http-client'
```

Comment:
I've used [`js-ipfs-ctl`](https://github.com/ipfs/js-ipfsd-ctl) to find out if there is a local ipfs daemon running, maybe check before going for infura?